src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -29,6 +29,20 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
     MTRDeviceStateUnreachable = 2
 };
 
+/**
+ * This bitmask is used to specify the type of logs requested from this device.
+ *
+ * The logs types are : End User Support, Network Diag and Crash logs.
+ */
+typedef NS_OPTIONS(NSUInteger, MTRDiagnosticLogTypes) {
+    MTRDiagnosticLogTypeEndUserSupport = 0, // End user support logs are requested
```

Comment:
How do I differentiate between "just network diagnostics logs" and "both network diagnostics and user support logs"?

This should be 1 << 0, and all the other bits shifted.

Reply:
this is my bad. I initially had it that way. i tried to stupidly match the enum

Reply:
done.